src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -443,6 +443,312 @@ public static ReadOnlyMemory<char> AsMemory(this string? text, Range range)
             return SpanHelpers.LastIndexOf<T>(ref MemoryMarshal.GetReference(span), span.Length, ref MemoryMarshal.GetReference(value), value.Length);
         }
 
+        /// <summary>Searches for the first index of any value other than the specified <paramref name="value"/>.</summary>
+        /// <typeparam name="T">The type of the span and values.</typeparam>
+        /// <param name="span">The span to search.</param>
+        /// <param name="value">The value for which to search.</param>
```

Comment:
Shouldn't this be `The value to avoid` like in the other overrides? (line 460, for example).

Reply:
Yeah, copy and paste error.